deps/unwind.mk

```diff
@@ -48,12 +48,12 @@ $(BUILDDIR)/libunwind-$(UNWIND_VER)/build-configured: $(SRCCACHE)/libunwind-$(UN
 	echo 1 > $@
 
 $(BUILDDIR)/libunwind-$(UNWIND_VER)/build-compiled: $(BUILDDIR)/libunwind-$(UNWIND_VER)/build-configured
-	$(CMAKE) --build $(dir $<)
+	$(MAKE) -C $(dir $<)
```

Comment:
I don't really understand the changes in #54538; `cmake --build` straight up doesn't work for me in this case, which seems to be true for other folks as well given that #55049 includes this same change. Likewise for the change to `deps/zlib.mk` below, the Make flags aren't valid for passing to CMake.

Reply:
I've opened #55091 with this fix.